device/dell/x86_64-dell_z9100_c2538-r0/Force10-Z9100-T0/sai.profile

```diff
@@ -0,0 +1 @@
+SAI_INIT_CONFIG_FILE=/etc/bcm/th-z9100-8x100-48x50G.config.bcm
```

Comment:
Just an observation, rest of the config files specify `G` as in `8x100G`

Reply:
ok will change accordingly